inc/xmath.h

```diff
@@ -447,6 +447,26 @@
 	}\
 }\
 
+/**
+ * @brief      Sets an identity matrix.
+ *
+ * @param      TYPE   The storage type for each element.
+ * @param      R      Number of rows in the matrix
+ * @param      C      Number of columns in each row.
+ * @param      MAT    The matrix in question.
+ */
+#define MAT_IDENTITY(TYPE, R, C, MAT)\
```

Comment:
```suggestion
#define MAT_IDENTITY(R, C, MAT)\
```
I don't think we need the TYPE here.